packages/arb-token-bridge-ui/tests/support/common.ts

```diff
@@ -159,3 +159,80 @@ export const visitAfterSomeDelay = (
   cy.wait(15_000) // let all the race conditions settle, let UI load well first
   cy.visit(url, options)
 }
+
+export const wait = (ms = 0): Promise<void> => {
+  return new Promise(res => setTimeout(res, ms))
+}
+
+export async function generateActivityOnChains({
+  parentChainProvider,
+  childChainProvider,
+  wallet
+}: {
+  parentChainProvider: Provider
+  childChainProvider: Provider
+  wallet: Wallet
+}) {
+  async function fundWalletEth(walletAddress, networkType: 'L1' | 'L2') {
```

Comment:
we already have a function like that in synpress config, cant we reuse?

Reply:
Good catch - updated